libs/non-integral/reference/non_integral.c

```diff
@@ -241,7 +241,7 @@ int ref_exp(mpz_t rop, const mpz_t x)
 int findE(const mpz_t x)
 {
   mpz_t x_, x__, temp_q, temp_r;
```

Comment:
These declarations of `temp_q` and `temp_r` are still here - is it on purpose?

Reply:
Will fix...

Reply:
resolved.